### PR TITLE
created company model, added validations and model tests closes #21

### PR DESCRIPTION
**Pull request by bhargavisatpathy**
Created company model with attributes: name, organization, title, state, city and email.
Added validations for presence and email format.
Wrote tests.

screen shot 2015-04-20 at 9 59 34 pm

### CODE REVIEW

File: spec/models/company_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                       'company.name@email.com',
+                       'company_name@email.com'
+                      ).for(:email)
+  end
+  it do
+    should_not allow_value('company@email',
+                           'company.com',
+                           'company'
+                          ).for(:email)
+  end
```
Comment (cluhring) [inline spec/models/company_spec.rb:21]: shoulda-matchers, huh?  looks clean/good.

**Comment by cluhring** [comment]
:penguin:

File: spec/models/company_spec.rb (hunk 1 above)
Comment (KristaANelson) [inline spec/models/company_spec.rb:21]: This is what I love about code reviews! Always learning! :star: :star:

File: app/models/company.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Company < ActiveRecord::Base
+  validates :name, :organization, :title, :state, :city, presence: true
+  validates :email, presence: true, length: { maximum: 255 },
+                    format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i, }
+
+end
```
Comment (KristaANelson) [inline app/models/company.rb:6]: Nice and clean validations! I'm interested to see how the layout comes back but do you think we should add in the number of hires they are committing to as well?

**Pull request merged**